### DISCUSSION TITLE
Received 500 response from login api endpoint

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [vercel/community#724] [Help] [answered]
I make next app with next-friebase-auth package and firebaseui-web package
All function works in local but didn't work in Netlify and Vercel.

Error: Received 500 response from login api endpoint
https://too1.netlify.app

**Comment by swarnava** [comment, marked as answer]
It looks like hosted in Netlify and I was able to successfully login. Are you still facing this issue?
